source/find_getmore_killcursors_commands.rst

```diff
@@ -468,7 +468,7 @@ getMore
 -------
 
 The **getMore** command replaces the **OP_GET_MORE** wire protocol message.
-The query flags passed to OP_QUERY for a getMore command MUST be slave_ok=True
+The query flags passed to OP_QUERY for a getMore command MUST be secondary_ok=True
```

Comment:
Since the same document uses camelCase above, would it make sense to use "secondaryOk" here and in the instances below?

Reply:
Since the original spec text used `slaveOk` and later `slave_ok=True`, I thought there was some significance to the use of snake_case. But I guess the reason was the original spec author probably worked on the Python driver. Changed to `secondaryOk=true` to maintain consistency with previous usages in the spec.